test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -8,8 +8,9 @@ n_monitor_nodes: 1
 
```

Comment:
Is it left over from the test ? should it be reverted ?

Reply:
Are you talking about the changes in this file?
No, this is intentional, I thought of changing this job to use it always.
But I hoped to get more nemesis to run in 3h.

This way we should have one job that always run the same and perform as some sort of sanity to detect regressions easily.